ambry-commons/src/main/java/com.github.ambry.commons/ResponseHandler.java

```diff
@@ -53,9 +53,17 @@ private void onServerEvent(ReplicaId replicaId, ServerErrorCode errorCode) {
         break;
       case Partition_ReadOnly:
         clusterMap.onReplicaEvent(replicaId, ReplicaEventType.Partition_ReadOnly);
-        //fall through
+        clusterMap.onReplicaEvent(replicaId, ReplicaEventType.Disk_Ok);
```

Comment:
is this all tested?

Reply:
tested

Reply:
Possible tests are added in `HelixClusterManagerTest`.